py3status/modules/getjson.py

```diff
@@ -68,28 +65,17 @@ class Py3status:
     def getjson(self):
         """
         """
-        response = {
-            'cached_until': self.py3.time_in(self.cache_timeout),
-        }
-
         try:
-            resp = self.py3.request(self.url, timeout=self.timeout)
-            status = resp.status_code == 200
-            resp = resp.json()
+            getjson = self.py3.request(self.url, timeout=self.timeout).json()
+            getjson = self.py3.flatten_dict(
+                getjson, delimiter=self.delimiter, intermediates=True)
         except self.py3.RequestException:
-            resp = None
-            status = False
-
-        if status:
-            response['full_text'] = self.py3.safe_format(
-                self.format,
-                self.py3.flatten_dict(
-                    resp, delimiter=self.delimiter, intermediates=True
-                )
-            )
-        else:
-            response['full_text'] = ''
-        return response
+            getjson = None
```

Comment:
This is interesting....

say I have format `info: {moo}`  when no data what do we wish to show?

`<nothing>`  <- what we show currently

`info: {moo}`  <- after this PR placeholders show as 'missing'

`info: ` <- I like this we make all placeholders `''` so they get hidden

what do you think?